### ISSUE TITLE
Cant install the build 

### ISSUE THREAD

**Issue by bouncer019** [Jire/Charlatano#503]
ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.

Please set the JAVA_HOME variable in your environment to match the
location of your Java installation.

I have java installed and the version 8u151
what can i do?

**Comment by Jire** [comment]
You need the Java Development Kit (JDK) installed, not just the Java Runtime.

**Comment by bouncer019** [comment]
yes and i have jdk installed, i followed all steps in tutorial etc... and still saying that, i tried the build already made and says java is not an executable batch

**Comment by Bonfire** [comment]
Make sure that you have your JAVA_HOME and PATH environment variables set correctly on your system.

For JAVA_HOME:
1. In Search, search for and then select: System (Control Panel)
2. Click the Advanced system settings link.
3. Click Environment Variables. In the section System Variables, find the JAVA_HOME environment variable and select it. Click Edit. If the JAVA_HOME environment variable does not exist, click New.
4. In the Edit System Variable (or New System Variable) window, specify the value of the JAVA_HOME environment variable. In our case, the specified value would be the location of your installed Java Development Kit (JDK). An example of this would be "C:\Program Files\Java\jdk1.6.0_02"
5. Click OK. Close all remaining windows by clicking OK.

For PATH:
1. In Search, search for and then select: System (Control Panel)
2. Click the Advanced system settings link.
3. Click Environment Variables. In the section System Variables, find the PATH environment variable and select it. Click Edit. If the PATH environment variable does not exist, click New.
4. In the Edit System Variable (or New System Variable) window, specify the value of the PATH environment variable. In our case, the specified value would be the bin folder as part of the JDK, or simply "%JAVA_HOME%\bin"
5. Click OK. Close all remaining windows by clicking OK.

Note that if the environment variables mentioned above are already created, you may need to append the specified values to the variable by using the semicolon symbol as a separator. On Windows 10, the variable appears as a list and the semicolon is unnecessary.
